cmd/upgrade/executor/cstor_spc.go

```diff
@@ -103,6 +103,18 @@ func (u *UpgradeOptions) RunCStorSPCUpgrade(cmd *cobra.Command) error {
 			fmt.Println(err)
 			return errors.Errorf("Failed to upgrade cStor SPC %v:", u.cstorSPC.spcName)
 		}
+	case "1.1.0-1.2.0":
+		fmt.Println("Upgrading to 1.2.0")
+		err := upgrade100to110.Exec(u.fromVersion, u.toVersion,
```

Comment:
Also, should we refactor this to keep it generic as we are upgrading multiple cases using same package. 

```
upgrade100to110 "github.com/openebs/maya/pkg/upgrade/1.0.0-1.1.0/v1alpha1"
```

Reply:
Should it be `upgrade100to120 "github.com/openebs/maya/pkg/upgrade/1.0.0-1.1.0/v1alpha1"`
Will change the directory to `github.com/openebs/maya/pkg/upgrade/1.0.0-1.2.0` in next PR